rosdep/python.yaml

```diff
@@ -6108,11 +6108,15 @@ python3-scipy:
   gentoo: [sci-libs/scipy]
   macports: [py37-scipy]
   opensuse: [python3-scipy]
+  ubuntu: [python3-scipy]
+python3-scipy-pip:
```

Comment:
>  Also fixed the existing `scipy` tags to follow the naming convention outlined in [/CONTRIBUTING.md@`master`](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md?rgh-link-date=2020-06-27T16%3A05%3A26Z)

I appreciate the effort to bring the rosdep database into conformity with our current contributing guidelines. To avoid breaking keys that may be actively in use we tend not to correct keys which don't conform until they are used only by platforms that are no longer supported.

In short, I think that adding the updated key is reasonable but I think we ought to preserve the existing osx via pip definition under the `python3-scipy` key since it could still be in use.

Reply:
no problem. I can revert that part